docs/sql-ref-syntax.md

```diff
@@ -19,7 +19,7 @@ license: |
   limitations under the License.
 ---
 
-Spark SQL is a Apache Spark's module for working with structured data.
+Spark SQL is an Apache Spark's module for working with structured data.
```

Comment:
Actually, just remove the 'a'. The copy/pasted text is elsewhere and has this problem; if there are any more occurrences you can fix them.